.github/CODEOWNERS

```diff
@@ -1,6 +1,5 @@
 *	@ricardozanini @krisv
 
-*.adoc	@hmanwani-rh
+*.adoc	@hmanwani-rh @radtriste @MarianMacik
```

Comment:
We will probably distribute this anyway across the team, but I'm OK with this.

Reply:
Once we get a bot, we can make this better. Let's approach this way for now.